config/config_test.go

```diff
@@ -351,3 +354,22 @@ func TestEnvProvider_Callbacks(t *testing.T) {
 	assert.NoError(t, p.RegisterChangeCallback("test", nil))
 	assert.NoError(t, p.UnregisterChangeCallback("token"))
 }
+
+func TestGetConfigFiles(t *testing.T) {
+	SetEnvironmentPrefix("TEST")
```

Comment:
You can use env.Override available in the testutils. 
``defer env.Override(t, <env key>, <temp value>()``